src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,28 @@
+package calculator
+
+class StringAddCalculator(
+    private val stringSeparator: Separator = StringSeparator()
+) {
+    fun add(expression: String?): Int {
+        if (expression.isNullOrBlank()) {
+            return 0
+        }
+        return getNumbers(expression).sum()
```

Comment:
kotlin 에서는 if가 문이 아니라 식이어서, 아래 처럼도 사용할 수 있어요 :)
참고 정도만 하셔도 좋습니다 😀
```suggestion
        return if (expression.isNullOrBlank()) 0 else getNumbers(expression).sum()
```